src/sfctl/custom_is.py

```diff
@@ -0,0 +1,88 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# -----------------------------------------------------------------------------
+
+"""Custom commands for the Service Fabric infrastructure service support"""
+
+def is_command(client, command_input, service_id=None, timeout=60):
+    """Invokes an administrative command on the given Infrastructure Service
+    instance.
+
+    For clusters that have one or more instances of the Infrastructure
```

Comment:
Don't add long docstings in the custom files, please add using the YAML format for help files under the `helps` folder.

Reply:
I think you mentioned it before, but I can't remember; why do we avoid putting it here? This is the same method that we get the docs if we were using the SDK, so this is much easier to maintain, since we can copy and paste things over directly, in case there are changes that need to be made to it.

Reply:
For example, I find myself having to manually redo some corrections I have made in swagger in the help files. If the specific case does not require formatting which cannot be produced outside of the help files, then it is much easier to bulk replace everything in the custom command file.

Reply:
- Readability for the function definitions
- YAML help strings features are a superset of docstring features
- Consistency with other Knack-based commands

Reply:
moved to help file.